exercises/kindergarten-garden/instructions.md

```diff
@@ -1,33 +1,26 @@
-# Description
+# Instructions
 
-Given a diagram, determine which plants each child in the kindergarten class is
-responsible for.
+Your task is to, given a diagram, determine which plants each child in the kindergarten class is responsible for.
 
-The kindergarten class is learning about growing plants.
-The teacher thought it would be a good idea to give them actual seeds, plant them in actual dirt, and grow actual plants.
+There are 12 children in the class:
+
+- Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry.
+
+Four different types of seeds are planted:
 
-They've chosen to grow grass, clover, radishes, and violets.
+- Grass ("G"), clover ("C"), radishes ("R"), and violets ("V")
```

Comment:
I've added the character encoding the plant this time.

Reply:
Would a table make sense here or is that overkill?

Reply:
🤷‍♂️ What do others do think?

Reply:
I'm for adding the table. It's not clear what those strings are for at this point since we don't have the examples yet.
If we don't want a table, we can do a sentence like `Grass is marked on the diagram by "G", clover by "C", radishes by "R", and violets by "V".`

Reply:
I've added a table.